packages/plugin-lighthouse/CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+# Contributing
+
+## Setup
+
+Make sure to install dependencies:
+
+```sh
+npm install
+```
+
+### Chrome path
+
+In this plugin we provide Lighthouse functionality exposed over the `lighthousePlugin`.
+To e2e test lighthouse properly we work with a predefined testing setup.
+
+On some OS there could be a problem finding the path to Chrome.
+
+We try to detect it automatically in the set-setup script.
+
+If no chrome path is detected the error looks like this: `Runtime error encountered: No Chrome installations found.`
+
+To prevent this from happening you have to provide the path manually in your `.env`:
+
+```bash
+CUSTOM_CHROME_PATH=/Applications/Google Chrome.app/Contents/MacOS/Google Chrome
+```
+
+We added consider this path in our `beforeAll` hook.
+
+```ts
+beforeEach(() => {
+  try {
+    vi.stubEnv('CHROME_PATH', getChromePath());
+  } catch (e) {
+    const customChromePath = (process.env as { CUSTOM_CHROME_PATH: string }).CUSTOM_CHROME_PATH;
```

Comment:
No type-casting is needed, the value of `process.env` properties is `string | undefined`, which you're already safely handling. I guess you came up against the `Property 'CUSTOM_CHROME_PATH' comes from an index signature, so it must be accessed with ['CUSTOM_CHROME_PATH']` TypeScript error? The fix is to use element access.

```suggestion
    const customChromePath = process.env['CUSTOM_CHROME_PATH'];
```